perun-openapi/openapi.yml

```diff
@@ -9028,42 +9088,67 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
-  /urlinjsonout/usersManager/changePassword/login:
+  /json/usersManager/changePassword/login:
     post:
       tags:
         - UsersManager
       operationId: changePasswordForLogin
       summary: Changes password for a user in specified login-namespace.
-      parameters:
-        - $ref: '#/components/parameters/login'
-        - $ref: '#/components/parameters/namespace'
-        - { name: newPassword, schema: { type: string }, in: query, required: true }
-        - { name: oldPassword, schema: { type: string }, in: query, required: false }
-        - { name: checkOldPassword, schema: { type: boolean }, in: query, required: false, description: "True if the oldPassword has to be checked. When omitted it defaults to false." }
       responses:
         '200':
           $ref: '#/components/responses/VoidResponse'
         default:
           $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: InputChangePasswordForLogin
+              description: "input for changePasswordForLogin"
+              type: object
+              required:
+                - login
+                - namespace
+                - newPassword
+              properties:
+                login: { $ref: '#/components/parameters/login' }
+                namespace: { $ref: '#/components/parameters/namespace' }
+                newPassword: { $ref: '#/components/parameters/password' }
+                oldPassword: { $ref: '#/components/parameters/password' }
+                checkOldPassword: { type: boolean }
```

Comment:
Maybe keep the original description here so that the default value is checkOldPassword is clear. What do you think?